src/amo/components/Errors/ErrorPage/index.js

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import makeClassName from 'classnames';
+import NestedStatus from 'react-nested-status';
+import PropTypes from 'prop-types';
+
+import './styles.scss';
+
+export default class ErrorPage extends React.Component {
+  static propTypes = {
+    children: PropTypes.node,
+    className: PropTypes.string,
+    code: PropTypes.number,
+    header: PropTypes.node,
+  };
+
+  render() {
+    const { children, className, header, code } = this.props;
+
+    return (
+      <NestedStatus code={code}>
+        <section
+          className={makeClassName('Card Errors Card--no-footer', className, {
+            'Card--no-header': !header,
+            NotAuthorized: code === 401,
+            NotFound: code === 404,
+            ServerError: code === 500,
+          })}
+        >
+          {header ? <header className="Card-header">{header}</header> : null}
```

Comment:
Why are you re-implementing `Card` here with this `header` and the following `div`, when you could just used `Card`?

Reply:
Tried to get rid of css classes after all, not so fruitful. Will be using `Card`